vue-thacer/src/assets/js/thacer-map-setup-search.js

```diff
@@ -34,9 +34,9 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
             let label = obj.properties.Pi ? 'Π' + obj.properties.Pi : obj.properties.ID
 
             document.getElementById('nonloc').innerHTML += [
-              '<a class="unlocalised-tag px-1 m-0 border border-white" href="ceram?ID=' +
+              '<a class="unlocalised-tag px-1 m-0 border border-white" href="#/ceram?ID=' +
                 obj.properties.ID +
-                '&ANA=8888880&INV=88880">' +
+                '">' +
```

Comment:
aaah now I see

Reply:
Well, I would still like to know what this is all about :D

Reply:
Trying to find the best way to explain, I think it is just about having to mention the ID in the routes and not ID + ANA + INV which are IDs like use for photo.